js/image/shared/ImageUploader.svelte

```diff
@@ -86,11 +89,20 @@
 			});
 			return;
 		}
+		upload_id = Math.random().toString(36).substring(2, 15);
```

Comment:
what is this `upload_id` used for?

Reply:
We send it over in the `/upload` request so that the backend can stream back the progress by upload id